Homework_13/task_2.py

```diff
@@ -0,0 +1,31 @@
+# Implement writer for proxy pattern from the lesson
+
+import os
+
+
+class FileWriter:
+    def write_file(self, filename, content):
+        with open(filename, 'w') as file:
+            file.write(content)
+        print(f"File '{filename}' has been written.")
+
+
+class FileWriterProxy:
+    def __init__(self):
+        self.file_writer = FileWriter()
+
+    def write_file(self, filename, content):
+        if self.is_allowed(filename):
+            self.file_writer.write_file(filename, content)
+        else:
+            print("Sorry, you are not allowed to write this file.")
```

Comment:
and where is proxy? you just not allow to write in some cases. I want you to think how to add proxy to writing in scope of file data